### ISSUE TITLE
vars example returns NotImplementedError: VarString does not support equality checks

### ISSUE THREAD

**Issue by toadies** [Titan-Systems/titan#124]
I am not able to use vars feature in python or yaml.

```
>>> from titan.blueprint import Blueprint
>>> from titan import var
>>> from titan.resources import Database
>>> # Deferred style
>>> 
>>> db = Database(name=var.foobar)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "venv/lib/python3.10/site-packages/titan/resources/database.py", line 164, in __init__
    if self._data.name != "SNOWFLAKE":
  File "venv/lib/python3.10/site-packages/titan/var.py", line 20, in __eq__
# Interpolation style

db = Database(name="db_{{var.fo    raise NotImplementedError("VarString does not support equality checks")
NotImplementedError: VarString does not support equality checks
>>> # Interpolation style
>>> 
>>> db = Database(name="db_{{var.foobar}}")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "env/lib/python3.10/site-packages/titan/resources/database.py", line 164, in __init__
    if self._data.name != "SNOWFLAKE":
  File "venv/lib/python3.10/site-packages/titan/var.py", line 20, in __eq__
    raise NotImplementedError("VarString does not support equality checks")
NotImplementedError: VarString does not support equality checks
```

**Comment by teej** [comment]
Oversight on my part, I'll get this fixed today.

**Comment by teej** [comment]
Fixed in [v0.10](https://github.com/Titan-Systems/titan/releases/tag/v0.10.0)
